src/xenia/base/filesystem_wildcard.cc

```diff
@@ -89,7 +89,9 @@ void WildcardEngine::PreparePattern(const std::string_view pattern) {
   }
   if (last != pattern.size()) {
     std::string str_str(pattern.substr(last));
-    rules_.push_back(WildcardRule(str_str, WildcardFlags::LAST));
+    rules_.push_back(
+        WildcardRule(str_str, last ? WildcardFlags::LAST
+                                   : WildcardFlags(true, true, false)));
```

Comment:
For clarity, maybe define a `FIRST_OR_LAST` up with the others and use that here.

Reply:
For now I will only add  ``FIRST_AND_LAST`` (or ``MATCH_STRING``/``MATCH_PATTERN``) as it is used. 
If there will be a need for different combinations then we can add them.